builder.py

```diff
@@ -0,0 +1,208 @@
+# Taken from SELU implementation from https://github.com/bioinf-jku/SNNs/blob/master/selu.py
+import os
+
+import tensorflow as tf
+import random
+'''
+Tensorflow Implementation of the Scaled ELU function and Dropout
+Taken from https://github.com/hunkim/DeepLearningZeroToAll
+'''
+import numbers
+from tensorflow.contrib import layers
+from tensorflow.python.framework import ops, tensor_shape, tensor_util
+from tensorflow.python.ops import math_ops, random_ops, array_ops
+from tensorflow.python.layers import utils
+
+from constants import *
+
+DIRNAME = os.path.dirname(os.path.realpath(__file__)) + '/saved_graphs'
```

Comment:
have this be a SeLuModel attribute where the default value is as specified here and may be modified depending on the user input.